test/conformance/service_test.go

```diff
@@ -364,38 +409,106 @@ func TestReleaseService(t *testing.T) {
 	}
 	revisions = append(revisions, names.Revision)
 
+	// Also verify traffic is in the correct shape.
+	desiredTrafficShape["latest"] = v1alpha1.TrafficTarget{
```

Comment:
For a different PR, but what do you think about making the latest, candidate, and current strings constants in service_types.go since they are part of the API spec.

Reply:
Sounds like a good idea. But this would change probably hundreds of files, so let's do it separately.